VRCFaceTracking.Core/Params/Data/UnifiedData.cs

```diff
@@ -83,6 +83,22 @@ public struct UnifiedExpressionShape
         public float Weight;
     }
 
+    public struct UnifiedHeadData
+    {
+        /// <summary>
+        /// Contains head rotation and position data.
+        /// Head rotation values should be normalized to a [-1, 1] range representing -90d to 90d rotation
+        /// Head position values should be normalized / capped to a [-1, 1] range and represent deviation from user origin point
```

Comment:
Not entirely sure if necessary but should we have a defined unit system for head position (eg. real world meters) for the position (I am assuming this is how most desktop apps send it though)?

Reply:
ifacialmocap gives you a sort of nebulous value, you're right that we may want this range to be normalized to some actual movement range though, so that different modules don't result in drastically different results